build.gradle

```
@@ -49,16 +50,33 @@ sourceSets {
   }
 }
 
+task preserveScripts(type: Copy) {
+     from "bin/rapidwright", "bin/rapidwright.bat"
+     into scriptsDir
+}
+
 task copyJars(type: Copy) {
     from configurations.api
     into 'jars'
     //testFixturesImplementation contains the output jar, as well as an implicit dependency on the task.
     //Filter to remove the jar itself, getFiles() to remove the implicit dependency
     from configurations.testFixturesImplementation.filter { !tasks.getByName("jar").outputs.files.contains(it) } .getFiles()
     into 'jars'
+    dependsOn preserveScripts
+
 }
 compileJava.dependsOn(copyJars)
```

```suggestion
}
compileJava.dependsOn(copyJars)
compileJava.dependsOn(preserveScripts)
```
for consistency/clarity.

---

I thought this wouldn't work because I thought the deletion was being triggered during the `copyJars` task, but I just tried it again and it appears to work.